src/pages/PostProtectedIndex.js

```diff
@@ -1,9 +1,40 @@
-import React from 'react'
+import React from "react"
+import { Card, CardImg, CardBody, CardTitle, Button, CardSubtitle } from "reactstrap";
+import { NavLink } from "react-router-dom"
 
-function PostProtectedIndex() {
-  return (
-    <div>PostProtectedIndex</div>
-  )
+
+const PostProtectedIndex = ({posts, currentUser}) => {
+    const currentPosts = posts.filter(post => post.user_id === 1)
```

Comment:
instead of hardcoding the id...dynamically give the currentUser primary key, so you can have the ability to use the logic with any user.